charts/app-stable-diffusion/templates/deployment.yaml

```
@@ -1,5 +1,5 @@
 {{- if .Values.api.enabled }}
-{{- $huggingfaceDir := "/home/worker/app/huggingface-cache" -}}
+{{- $huggingfaceDir := "/mnt/models/Diffusers" -}}
```

```suggestion
{{- $huggingfaceDir := .Values.api.env.HUGGINGFACE_HUB_CACHE -}}
```

or maybe we could just drop it?